### DISCUSSION TITLE
Request for Adding Functionality to Set Headers in Tables and Help with Datatable2 Widget Integration

### DISCUSSION THREAD

**Discussion by TalentumSebasG** [flet-dev/flet#3687] [Ideas]
I hope you are doing well.

I would like to propose an enhancement for table functionality in our application and request assistance with a technical integration. Below I detail both points:

Set Headers in Tables:

I think it would be very beneficial to implement an option to set table headers when scrolling. This feature would facilitate data visualization by keeping column names always visible, especially in long tables that require vertical scrolling.

Provide the ability to enable or disable this functionality according to user preferences.

![datatable1](https://github.com/user-attachments/assets/f042f62c-4c78-4e84-8dae-f2bf93aa2dfe)

![datatable2](https://github.com/user-attachments/assets/c570984d-b659-464d-bc3b-2d06c9729e26)

Assistance with Flutter datatable2 Widget Integration:

I read that it is possible to integrate Flutter's datatable2 widget with our application, however, I have not been able to perform the integration following the guidance provided in the Flet documentation. Therefore, I would appreciate if you could provide assistance in this process or point me to an alternative solution.

**Comment by ndonkoHenri** [comment]
Here are some links to explanations on how to integrate a Flutter package in Flet:
- https://youtube.com/playlist?list=PLHYEoLmkidc1YrgREPrJvhr_JEv00EFP8&si=d5kYvRUBwP3q_bur
- https://flet.dev/docs/extend/integrating-existing-flutter-packages
